src/frontend/js/index.tsx

```diff
@@ -7,15 +7,26 @@
 
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import { IntlProvider } from 'react-intl';
+import { addLocaleData, IntlProvider } from 'react-intl';
+import en from 'react-intl/locale-data/en';
+import es from 'react-intl/locale-data/es';
+import fr from 'react-intl/locale-data/fr';
```

Comment:
seems weird to hardcode all languages here? Shouldn't it be in a setting so it can scale to (much) more languages?